cmd/influxd/inspect/verify_seriesfile.go

```diff
@@ -9,10 +9,90 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/influxdata/influxdb/v2/logger"
 	"github.com/influxdata/influxdb/v2/tsdb"
+	"github.com/spf13/cobra"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
+type args struct {
+	dir        string
+	db         string
+	seriesFile string
+	verbose    bool
+	concurrent int
+}
+
+func NewVerifySeriesfileCommand() *cobra.Command {
+	var arguments args
+	cmd := &cobra.Command{
+		Use:   "verify-seriesfile",
+		Short: "Verifies the integrity of series files.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SetOut(os.Stdout)
+
+			config := logger.NewConfig()
+			config.Level = zapcore.WarnLevel
+			if arguments.verbose {
+				config.Level = zapcore.InfoLevel
+			}
+			log, err := config.New(os.Stderr)
+			if err != nil {
+				return err
+			}
+
+			v := NewVerify()
+			v.Logger = log
+			v.Concurrent = arguments.concurrent
+
+			var db string
+			if arguments.seriesFile != "" {
+				db = arguments.seriesFile
+			} else if arguments.db != "" {
+				db = filepath.Join(arguments.dir, arguments.db, "_series")
+			}
+			if db != "" {
+				_, err := v.VerifySeriesFile(db)
+				return err
+			}
+
+			dbs, err := ioutil.ReadDir(arguments.dir)
+			if err != nil {
+				return err
+			}
+
+			for _, db := range dbs {
+				if !db.IsDir() {
+					continue
+				}
+				_, errNew := v.VerifySeriesFile(filepath.Join(arguments.dir, db.Name(), "_series"))
+				if errNew != nil {
+					v.Logger.Error("Failed to verify series file",
+						zap.String("filename", fmt.Sprintf(filepath.Join(arguments.dir, db.Name(), "_series"))),
```

Comment:
You could pull the `filepath.Join` into a local var after the `db.IsDir` check, and reuse it across the verify call and this log.

I also don't think the `Sprintf` is needed here.